src/Core/src/Handlers/Button/NET8/ButtonHandler2.Standard.cs

```diff
@@ -0,0 +1,8 @@
+using System;
+
+namespace Microsoft.Maui.Handlers
+{
+	public partial class ButtonHandler2 : ButtonHandler
```

Comment:
This is probably wrong now that I'm thinking about it because this doesn't let us deprecate the `ButtonHandler` so would probably need to pull all the code out.